tests/components/komfovent/test_config_flow.py

```diff
@@ -0,0 +1,191 @@
+"""Test the Komfovent config flow."""
+from unittest.mock import AsyncMock, patch
+
+import komfovent_api
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.komfovent.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult, FlowResultType
+
+from tests.common import MockConfigEntry
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
```suggestion
```
I think you can remove this (since you're already using `mock_setup_entry` in the test parameters)